### PR TITLE
Task/report sickness now

### PR DESCRIPTION
**Pull request by bastianh**
contains: https://tasks.pxp-x.com/browse/CTAA-1622

### CODE REVIEW

File: CoronaContact/Logic/QuarantineTimeController.swift [hunk 1]
```diff
@@ -176,7 +176,36 @@ class QuarantineTimeController {
             return completion(.inProgressIndefinitely)
         }
 
+        var endOfQuarantines = [QuarantineEnd]()
+
         // TODO: add new calculation
```
Comment (finnsch) [inline CoronaContact/Logic/QuarantineTimeController.swift:181]: Is the TODO comment still necessary?

File: CoronaContact/Scenes/Debug/DebugViewController.swift [hunk 2]
```diff
@@ -78,6 +79,10 @@ class DebugViewController: UIViewController, StoryboardBased, ViewModelBased, Re
         viewModel?.revokeAttestedSick()
     }
 
+    @IBAction func moveSickreportBackADay(_ sender: Any) {
+        viewModel?.moveSickreportBackADay()
```
Comment (finnsch) [inline CoronaContact/Scenes/Debug/DebugViewController.swift]:
```suggestion
        viewModel?.moveSickReportBackADay()
```

File: CoronaContact/Views/QuarantineNotificationView.swift [hunk 3]
```diff
@@ -185,6 +185,15 @@ class QuarantineNotificationView: UIView, NibOwnerLoadable {
         handleClose?()
     }
 
+    func addLabel(title: String) {
+        buttonsStackView.isHidden = false
+        let label = TransLabel()
+        label.numberOfLines = 0
+        label.attributedText = title.set(style: "body")
```
Comment (finnsch) [inline CoronaContact/Views/QuarantineNotificationView.swift]:
```suggestion
        label.attributedText = title.set(style: StyleNames.body.rawValue)
```

**Review by finnsch — CHANGES_REQUESTED** [review]

File: CoronaContact/Logic/QuarantineTimeController.swift (hunk 1 above)
Comment (bastianh) [reply CoronaContact/Logic/QuarantineTimeController.swift:181]: it misses the quarantine from contacts .)

Comment (bastianh) [reply CoronaContact/Logic/QuarantineTimeController.swift:181]: it misses the quarantine time from exposures

**Review by finnsch — APPROVED** [review]

**Pull request merged**